src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,51 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <div
+    className="card"
+    data-cy="Movie"
+  >
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          // eslint-disable-next-line
+          src={movie.imgUrl}
+          alt="Film logo"
```

Comment:
Alt attribute for image tag is not descriptive. It should describe the content of the image. Replace 'Film logo' with something like {movie.title} logo.